src/components/Button/Button.jsx

```diff
@@ -83,7 +83,7 @@ const Button = ({
       disabled={disabled}
       type={type}
       fullWidth={fullWidth}
-      loading={loading}
+      $loading={loading}
```

Comment:
Ooh, haven't seen this before! 

So, we're currently using a beta version of the ui library because there were react mismatch issues. Hopefully the beta will get published soon (it's waiting on cross-team PR reviews) 🤞 In the meantime, any non-beta versions will break publish. It may make sense to wait until it gets published before doing this change.

Reply:
Yeah, no rush TBH. Thanks for the ⏩ review though. 😄